assets/pages/bundles-landing/components/Introduction.jsx

```diff
@@ -3,19 +3,52 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import type { Participations } from 'helpers/abtest';
 
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+    abTests: Participations,
+}
 
+type HeaderCopy = {
+    line1: string,
+    line2: string,
+}
 // ----- Component ----- //
 
-export default function Introduction() {
+export default function Introduction(props: PropTypes) {
+
+  const headerCopyOptions: {[variant: string]: HeaderCopy} = {
+    control: {
+      line1: 'help us deliver the',
+      line2: `independent journalism the world${String.fromCharCode(160)}needs`,
+    },
+    numberOfSupporters: {
+      line1: 'Over 800,000 readers now help fund the Guardian',
```

Comment:
From what @superfrank tells me, `the Guardian` usually needs a non-breaking space too =)